File: API/Models/Folha.cs
using System;

namespace API.Models
{
    public class Folha
    {
        //Construtor
        public Folha() => CriadoEm = DateTime.Now;

        //Atributos
        public int Id { get; set; }
        public string quantHoras { get; set; }
        public string valorHoras { get; set; }
        public string mes { get; set; }
        public string ano { get; set; }
        public Funcionario Funcionario { get; set; }
        public int FuncionarioId { get; set; }
        public DateTime CriadoEm { get; set; }
    }
}

File: FRONT/src/app/models/folha.ts
import { Funcionario } from "./funcionario";

export interface Folha {
    id?: number;
    quantHoras: string;
    valorHoras: string;
    mes: string;
    ano: string;
    criadoem?: string;
    funcionarioId: number;
    funcionario?: Funcionario;
}
